core/src/main/java/com/amplitude/core/utilities/EventsFileManager.kt

```diff
@@ -76,7 +76,9 @@ class EventsFileManager(
         val fileList = directory.listFiles { _, name ->
             name.contains(apiKey) && !name.endsWith(".tmp")
         } ?: emptyArray()
-        return fileList.map {
+        return fileList.sortedBy { it ->
```

Comment:
Question, why do we need to sort the files based on the index?

Reply:
The reason for this is the default order is not the asc order, wants to align them to keep the order.